### ISSUE TITLE
High security bug in mygov.in

### ISSUE THREAD

**Issue by Onyx24** [nic-delhi/AarogyaSetu_Android#346]
Hi I found a high security bug in mygov.in I sent the bug report to your security email. Please check..

**Comment by riderattitude** [comment]
I don't think so it wouldn't be rewardable as they are looking up for inApp bug, though the app is hybrid, the above link seems external link and nothing to do with the app

**Comment by Onyx24** [comment]
Hi yes, but they need to fix this asap.

**Comment by sreejeet** [comment]
If you have sent an email to them, then you don't need to raise an issue here. Your response will be provided in your email. Please close this issue.
